dbt_copilot_helper/utils/aws.py

```diff
@@ -52,6 +52,13 @@ def get_aws_session_or_abort(aws_profile: str = None) -> boto3.session.Session:
             fg="red",
         )
         exit()
+    except botocore.exceptions.TokenRetrievalError:
+        click.secho(
+            "The SSO Token associated with this profile has expired.  "
+            "To refresh this SSO session run aws sso login with the corresponding profile",
```

Comment:
Just a bit to make it more obvious this is a command.
```suggestion
            "To refresh this SSO session run `aws sso login` with the corresponding profile",
```